Soccar [Physics]/src/nl/soccar/physics/models/CarPhysics.java

```diff
@@ -104,11 +103,17 @@ public void step() {
 
     @Override
     public void reset() {
-        body.setLinearVelocity(new Vec2(0.0F, 0.0F));
-        body.setAngularVelocity(0.0F);
-        body.setTransform(originalPos, originalDegree);
+        setPosition(originalPos.x, originalPos.y, originalDegree, 0, 0, 0);
+    }
+
+    public void setPosition(float x, float y, float degree, float linearVelocityX, float linearVelocityY, float angularVelocity) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)